.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,76 @@
+name: Test Emscripten/Pyodide build
+
+on:
+# TODO: refine when this workflow should run when this
+# is ready for use or before merging
+  pull_request:
+  push:
+  workflow_dispatch:
+
+env:
+  FORCE_COLOR: 3
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build-wasm-emscripten:
```

Comment:
Could you add this job in `.github/workflows/unit-tests.yml` instead?

Reply:
Sure!